### PR TITLE
Added array configmap params for powermax

### PR DESCRIPTION
**Pull request by harishp8889**
#### Is this a new chart?

No

#### What this PR does / why we need it:
Added array configmap for CSI-Powermax, This is essential as part of this PR: https://github.com/dell/csi-powermax/pull/318

#### Which issue(s) is this PR associated with:

- #Issue_Number
https://github.com/dell/csm/issues/1543
#### Checklist:

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [ ] Chart Version bumped
- [x] Variables are documented in the chart README.md
- [ ] Title of the PR starts with the chart name (e.g. `[charts_dir/mychartname]`) if applicable

### CODE REVIEW

**Review by bpjain2004 — APPROVED** [review]

File: charts/csi-powermax/templates/controller.yaml [hunk 1]
```diff
@@ -546,6 +550,9 @@ spec:
         - name: tls-secret
           secret:
             secretName: {{ .Values.csireverseproxy.tlsSecret }}
+        - name: powermax-array-config
```
[the remark this answers is not in the document]
Comment (harishp8889) [reply charts/csi-powermax/templates/controller.yaml:547]: The user has to create and yes we will have to update the CSM docs

File: charts/csi-powermax/values.yaml [hunk 2]
```diff
@@ -101,7 +130,7 @@ defaultFsType: ext4
 #  IfNotPresent: Only pull the image if it does not already exist on the node.
 #  Never: Never pull the image.
 # Default value: None
-imagePullPolicy: IfNotPresent
```
Comment (tdawe) [inline charts/csi-powermax/values.yaml:104]: Default value of IfNotPresent should remain for customers so they don't always pull this image

Comment (harishp8889) [reply charts/csi-powermax/values.yaml]: changed

**Review by xuluna — APPROVED** [review]

**Review by meggm — APPROVED** [review]
LGTM

**Review by bpjain2004 — APPROVED** [review]

**Pull request merged**